views/item_list.pug

```diff
@@ -13,4 +13,23 @@ block content
     else
       li There are no #{type}s to display.
 
-  a(href='/edit/' + type + '/create') Create #{type}
\ No newline at end of file
+  a(href='/edit/' + type + '/create') Create #{type}
+  <br>
+
+  //- if type === 'preset'
```

Comment:
oh missed this on my initial look, but why do we have this block of commented out code to have the options as buttons. I think we should stick with the link for now and shouldn't leave a block of commented out code here.

Reply:
These are the download button code commented out. I kept it because you initially told me to make buttons but I didn't find them visually pleasing so I kept the button  code just in case you wanted it to be button. I'll delete it now that we agreed that having links are fine.